Makefile

```diff
@@ -91,6 +127,31 @@ push:
 		$(DOCKER) push $(REGISTRY)/$(TARGET):latest; \
 	fi
 
+push_manifest:
+	/go/bin/manifest-tool push from-args --platforms $(PLATFORMS) --template $(REGISTRY)/$(TARGET)-ARCH:$(VERSION) --target  $(REGISTRY)/$(TARGET):$(VERSION)
+
+push: pre container
+	for arch in $(LINUX_ARCH); do \
+		$(MAKE) push_images TARGET="sonobuoy-$$arch"; \
```

Comment:
`An image does not exist locally with the tag: gcr.io/heptio-images/sonobuoy-amd64`

When should these be created?